### ISSUE TITLE
[Release] [1.6.0] Release tasks

### ISSUE THREAD

**Issue by ddimatos** [ansible-collections/ibm_zos_core#733]
## Release Requirements Checklist
This checklist is a chronological ordering of the release tasks to complete for an IBM Ansible z/OS core collection. This template is for a GA release that will release to Ansible Automation Platform and Galaxy as well.

By this point the `stagging-v<version>` had been code frozen and releases tasks must only be meta data related, for example, updating galaxy.yml, runtime.yml, generating module doc if needed, release notes etc.

You should be checking off each task as it completes.

## General workflow
This a subsequent release (not a patch) thus these releases tasks will eventually be merged into`main` then back into dev to allow for the meta data updates to propagate as well as the release tag. 
TBD 

### (1) Technical Writer Tasks
- [x] Ensure all technical writer Git issues are resolved for this release

### (2) Pull Request Tasks for `release-tasks-v<version>` branch:
- [x] Update `galaxy.yml`
- [x] Update `meta/runtime.yml`
- [x] Update `meta/ibm_zos_core_meta.yml`
- [x] Update `README.md`
- [x]  Update `.ansible-lint` config , keep it in synch with the `galaxy.yml` `build_ignore` entries. 
- [x] Review copyrights are updated as needed, e.g. current year if changes are made
- [x] Generate module docs (restructured text - RST) and check it into the release branch
- [x] Add a change log summary fragment in the `/changelog` directory for this release:
       - Create a file called "v\<major\>.\<minor\>.\<patch\>_summary.yml", eg `v1.6.0_summary.yml`
       - Place it under `ibm_zos_core/changelogs/fragments/`
       - Add contents (update the date, don't go past the current day):
```
release_summary: |
  Release Date: '2023-006-23'
  This changelog describes all changes made to the modules and plugins included
  in this collection. The release date is the date the changelog is created.
  For additional details such as required dependencies and availability review
  the collections `release notes <https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html>`__
```
- [x] Generate CHANGELOG.rst
    - `antsibull-changelog lint`
    - `antsibull-changelog release` or `antsibull-changelog release -v`
- [x] Create release notes (`docs/source/release_notes.rst`)
- [x] Review all module source imports; ensure sure no new imports are subject to license discrepancies 

### (3) Quality assurance
- [x] Run a Mend scan and check the results into the designated repository
   - Note: Doing this on the  `release-tasks-v<version>` branch provides opportunity to make last minute changes if needed.
- [x] Run a Bandit scan and check the results into the designated repository
- [x] Ansible-lint validation , manual step at the moment, choose from one of the below to run ansible-lint:
    - ./ac --ac-lint
    - ./ac --venv-start
       - ansible-lint -c .ansible-lint --profile production
- [x] Sanity tests, use the pipeline option and choose to run only the sanity tests by checking `ANSIBLE_TEST_ONLY` which specifies only Ansible Certification tests and security scan run, meaning no functional tests run."
- [x] Run the galaxy importer (currently a manual step as its not part of the pipeline yet)
     - [x] `python -m galaxy_importer.main ibm-ibm_zos_core-<version>.tar.gz`. 
     - [x] Check output log for any errors/issues. For success, look for the following msg:
        ```
         Collection loading complete
         Importer processing completed successfully
         ```
- [x] Full pipeline regression at 100% success on either `release-tasks-v<version>` or `stagging-v<version>`:
     - [x] Certify on IBM Enterprise Python versions 3.11  or latest (with the latest PTF for env var PYTHONSTDINENCODING)
     - [x] Certify on IBM Enterprise Python and new environment var PYTHONSTDINENCODING
     - [x] Certify on ZOAU 1.2.2 or latest
     - [x] Certify on `ansible-core v2.15.x` or latest supported by Ansible Automation Platform
     - [x] Ensure `RUN_ALL_TESTS` is selected

### (5) Certification and validation tasks
- [x] Build the image from `release-tasks-v<version>` branch, unpack it up, inspect the contents. Ensure only the required files are contained in the package. 
- [x] Test collection quality with an upload to the IBM HCF development Galaxy Server by uploading the build from branch `release-tasks-v<version>.
- [x] Open a pull request for `release-tasks-v<version>` against `staging-v<version>`, wait for reviewers.
- [x] Upon pull request completion, merge the PR  for `release-tasks-v<version>` into `staging-v<version>`
- [x] Test collection quality again with a new build from `staging-v<version>` and upload to the Ansible development Galaxy Server

### (6) GitHub
- [x] Open a pull request for `staging-v<version>` against `main`
   - Once the PR has been approved, merge the PR but **DO NOT** delete the `staging-v<version>`, you will need the `staging-v<version>` branch soon enough.
- [x] Create a [draft release](https://github.com/ansible-collections/ibm_zos_core/releases/new) in GitHub, for:
    - "Choose a tag", enter `v<version>`, eg `v1.6.0`
    - "Target" , enter `main`
    - "Release title" , enter `release-v<version>`, eg `release-v1.6.0`
    - "Describe this release", enter the contents created for `doc/source/release_notes.rst`, you may need to remove trailing `_` from the RST you paste since the description is supposed to be Markdown.
  

### (7) Release tasks (**In this order**)
- [x] Upload to [AAP](https://console.redhat.com/ansible/automation-hub)
- [ ] Only after AAP upload is live move forward on the remaining tasks: 
    - [x] Upload to [Galaxy](https://galaxy.ansible.com/ibm/ibm_zos_core)
    - [x] Merge pull request for `staging-v<version>` against `main` from bullet (6)
    - [x] Complete the Github release (tag) opened in bullet (6)
    - [x] Open a Git issue in [z_ansible_collections_doc](https://github.com/IBM/z_ansible_collections_doc/issues) to have [Red Hat® Ansible Certified Content for IBM Z docs](https://ibm.github.io/z_ansible_collections_doc/index.html) generated
    - [ ] Merge `stagging-v<version>` into `dev` to ensure any fixes and meta are put into `dev`.

### (8) Log collection
- [x] Copy any additional playbook or manual tests performed into the release folder
- [x] Copy the pipeline log from Jenkins into the release folder
- [x] Copy the Automation Hub import log into the release folder
- [x] Copy the Galaxy import log in the release folder
- [x] Copy the git log (`git log --pretty=oneline`) into the release folder

**Comment by ddimatos** [comment]
```
(venv-2.15) ./ac --ac-version
-----------------------------------------------------------------------
Ansible collection version installed on this controller.
-----------------------------------------------------------------------
 1.6.0
```

**Comment by ddimatos** [comment]
```
(venv-2.15) python -m galaxy_importer.main ibm-ibm_zos_core-1.6.0.tar.gz
Importing with galaxy-importer 0.4.10
Getting doc strings via ansible-doc
Finding content inside collection
Loading module zos_data_set
Loading module zos_ping
Loading module zos_encode
Loading module zos_mount
Loading module zos_job_output
Loading module zos_operator_action_query
Loading module zos_find
Loading module zos_mvs_raw
Loading module zos_backup_restore
Loading module zos_lineinfile
Loading module zos_copy
Loading module zos_volume_init
Loading module zos_apf
Loading module zos_tso_command
Loading module zos_job_query
Loading module zos_job_submit
Loading module zos_operator
Loading module zos_gather_facts
Loading module zos_fetch
Loading module zos_blockinfile
Loading module_utils ickdsf
Loading module_utils better_arg_parser
Loading module_utils mvs_cmd
Loading module_utils zoau_version_checker
Loading module_utils backup
Loading module_utils system
Loading module_utils encode
Loading module_utils job
Loading module_utils vtoc
Loading module_utils zos_mvs_raw
Loading module_utils copy
Loading module_utils ansible_module
Loading module_utils file
Loading module_utils dd_statement
Loading module_utils data_set
Loading module_utils import_handler
Loading action zos_ping
Loading action zos_copy
Loading action zos_job_submit
Loading action zos_fetch
Loading filter wtor
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.16.txt with 35 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.14.txt with 35 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.15.txt with 35 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.11.txt with 85 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.12.txt with 85 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.13.txt with 35 statement(s)
WARNING: Ignore files skip ansible-test sanity tests, found ignore-2.9.txt with 84 statement(s)
Collection loading complete
Importer processing completed successfully
```

**Comment by ddimatos** [comment]
At this time AAP is using ansible-lint v6.14.3 over our v6.17.1.

Lint is showing (below) as result of conscious decision to support 2.9 until a deprecation strategy is put forward for all ansible z collections. 

```
meta/runtime.yml:1: meta-runtime[unsupported-version]: requires_ansible key must be set to a supported version.
```

ansible-lint is also showing a number of known issues which come from the ignore-<version>.txt files. These ignore entries are a result of a competing parser (better-arg-parser) this collection is using tailored for z/OS and because the better-arg-parser uses the same module arg spec, these ignore.txt entries had to be made which will soon  no longer be permitted. 

These ansible-lint errors cause no functional issues but over time will result in a certification failure. Red Hat partner engineering has been made aware of them and will be overlooked in the near term, and we are planning to address them starting Qq 2023, see issue #811 

```
tests/sanity/ignore-2.11.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:24: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 26, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:27: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 27, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:30: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 30, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:33: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 33, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:40: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 40, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:41: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 41, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:56: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 56, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:57: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 57, which is not a permitted ignore.
tests/sanity/ignore-2.11.txt:58: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 58, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:24: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 26, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:27: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 27, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:30: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 30, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:33: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 33, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:40: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 40, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:41: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 41, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:56: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 56, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:57: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 57, which is not a permitted ignore.
tests/sanity/ignore-2.12.txt:58: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 58, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:2: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 2, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:9: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 9, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:11: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 11, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:17: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 17, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:22: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 22, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:24: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.13.txt:32: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 32, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:2: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 2, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:9: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 9, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:11: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 11, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:17: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 17, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:22: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 22, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:24: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:32: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 32, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:2: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 2, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:9: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 9, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:11: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 11, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:17: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 17, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:22: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 22, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:24: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:32: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 32, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:2: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 2, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-default-does-not-match-spec at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:9: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 9, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:11: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 11, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:17: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 17, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:22: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 22, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:24: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 24, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:31: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 31, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:32: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 32, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:16: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 16, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:23: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 23, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:25: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 25, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 26, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:29: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 29, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:30: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 30, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:32: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 32, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:39: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 39, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:40: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 40, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:55: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 55, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:56: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 56, which is not a permitted ignore.
tests/sanity/ignore-2.9.txt:57: sanity[cannot-ignore]: Ignore file contains pylint:catching-non-exception at line 57, which is not a permitted ignore.
Read documentation for instructions on how to ignore specific rule violations.

                         Rule Violation Summary
 count tag                               profile    rule associated tags
     1 meta-runtime[unsupported-version] shared     metadata
    98 sanity[cannot-ignore]             production idiom

Failed: 99 failure(s), 0 warning(s) on 64 files. Profile 'production' was required, but only 'safety' profile passed. Rating: 3/5 star
```

**Comment by ddimatos** [comment]
Regression summary:
```
====================== 1077 passed in 19262.90s (5:21:02) ======================
```
